New Checkout Drop-in v1.4.0-beta1 (#254)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/payment-methods.mdx

     */
    Handlers?: PaymentMethodsHandlerSlot;
    Methods?: PaymentMethodsSlot;
  } & TitleProps['slots'];
  UIComponentType?: UIComponentType;
}
```

<li>The `setOnChange` property has been _deprecated_ and will be removed in future versions. It has been replaced by the following one: `slots.Methods.<payment-method-code>.setOnChange`.</li>

> <li>The `UIComponentType` property is a string containing the name of the UI component type to be used as selector for each payment method. By now, the UI components available are: 'ToggleButton' or 'RadioButton'.</li>

Reviewer: ```suggestion
<li>The `UIComponentType` property is a string containing the name of the UI component type to be used as a selector for each payment method. The UI components available are: 'ToggleButton' or 'RadioButton'.</li>
```

<li>The `slots` property is an object containing the following properties:</li>

#### Title property (**)

> The `Title` property is inherited as a result of merging the `slots` from `PaymentMethodsProps` and `TitleProps`; this is a handler used to render a customized title content.

Reviewer: ```suggestion
The 'Title' property is an inherited property; it exposes a managed handler to render a custom title.
```

  },
})($paymentMethods),
```

## Example 5: Render with customized title and radio button as selector

> The following example renders the `PaymentMethods` container on a checkout page, displaying a customized title and showing radio buttons instead of toggle buttons as selector for each payment option:

Reviewer: ```suggestion
The following example renders the `PaymentMethods` container on a checkout page to display a custom title and radio buttons instead of toggle buttons for selecting the payment options:
```
